packages/destination-actions/src/destinations/the-trade-desk-crm/functions.ts

```diff
@@ -100,7 +104,7 @@ export async function processPayload(input: ProcessPayloadInput) {
 function extractUsers(payloads: Payload[]): string {
   let users = ''
   payloads.forEach((payload: Payload) => {
-    if (!payload.email) {
+    if (!payload.email || !validateEmail(payload.email, payload.pii_type)) {
```

Comment:
Nit: Verifying the email exists and then validating it seems redundant. The validation method should cover the presence of the email as well.
```suggestion
    if (!validateEmail(payload.email, payload.pii_type)) {
```